moshi/moshi/modules/transformer.py

```diff
@@ -625,14 +646,24 @@ def __init__(
                     gating, d_model, dim_feedforward, **factory_kwargs
                 )
 
+        self.cross_attention: StreamingMultiheadAttention | None = None
+        if cross_attention:
+            self.cross_attention = StreamingMultiheadAttention(
+                cross_attention=True, **attn_kwargs, **factory_kwargs)  # type: ignore
+            self.norm_cross = nn.LayerNorm(d_model, eps=1e-5, **factory_kwargs)  # type: ignore
```

Comment:
Will this always be a layer-norm or should we make this a config field and use `create_norm_fn`?

Reply:
In the training code it is always layer norm... it is not consistent with the rest but that's how the models were trained. I can still add a parameter for future use though.

Reply:
Makes sense, up to you if you want to put an additional parameter vs keep things in sync with the training side.

Reply:
In the interest of speed I'll keep it like that until we need it, just added a comment.